go/kbfs/libkbfs/md_util.go

```diff
@@ -672,5 +704,23 @@ func decryptMDPrivateData(ctx context.Context, codec kbfscodec.Codec,
 		return PrivateMetadata{}, err
 	}
 
+	var obfuscator data.Obfuscator
+	if keyedForDevice {
+		secret, err := getMDObfuscationSecret(ctx, keyGetter, rmdWithKeys)
+		if err != nil {
+			return PrivateMetadata{}, err
+		}
+		obfuscator = makeMDObfuscatorFromSecret(secret, mode)
+	}
+	for _, op := range pmd.Changes.Ops {
+		// Add a temporary path with an obfuscator.
```

Comment:
I don't understand what's going on here.

Reply:
When we deserialize the ops from the raw byte buffer of the MD object, they don't have proper `data.Path`s set in them yet -- that's an unexported field.  That used to not be a big deal, but now the path is required for obfuscation, so here I'm just making sure that they all have one.  In places where a perfectly-accurate path is required (like in conflict resolution), the code there will add one. So here's another place where the level of the obfuscator might be wrong and result in inconsistent conflict suffixes.  I'll add a comment clarifying this.